src/lib/terraform.ts

```diff
@@ -0,0 +1,19 @@
+import axios, { AxiosInstance, AxiosRequestConfig, AxiosResponse} from 'axios'
+
+export interface TerraformOptions {
+  token: string,
+  host: string,
+}
+
+export {AxiosInstance as TerraformInstance}
+export {AxiosResponse as TerraformResponse}
+
+export function terraform ({ token, host = 'app.terraform.io' } : TerraformOptions): AxiosInstance {
```

Comment:
Capitalize constructors

Reply:
And given that the module's name is `terraform` it seems like this should be something else describing the type of Terraform thing

https://blog.golang.org/package-names

> **Avoid repetition.** Since client code uses the package name as a prefix when referring to the package contents, the names for those contents need not repeat the package name. The HTTP server provided by the http package is called Server, not HTTPServer. Client code refers to this type as http.Server, so there is no ambiguity.

`Client` works here